.changeset/honest-states-repeat.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-home': minor
+---
+
+Added optional title prop to customHomePageGrid and improved the example app homepage to add the custom one
```

Comment:
```suggestion
Added optional title prop to `customHomePageGrid`
```

We don't ship the example app and as this gets rolled up in the Home plugin's changelog it won't make a lot of sense in that context.